be/src/runtime/primitive_type.h

```diff
@@ -212,11 +213,13 @@ VALUE_GUARD(PrimitiveType, Decimal64PTGuard, pt_is_decimal64, TYPE_DECIMAL64)
 VALUE_GUARD(PrimitiveType, Decimal128PTGuard, pt_is_decimal128, TYPE_DECIMAL128)
 VALUE_GUARD(PrimitiveType, DecimalPTGuard, pt_is_decimal, TYPE_DECIMAL32, TYPE_DECIMAL64, TYPE_DECIMAL128)
 VALUE_GUARD(PrimitiveType, SumDecimal64PTGuard, pt_is_sum_decimal64, TYPE_DECIMAL32, TYPE_DECIMAL64)
+VALUE_GUARD(PrimitiveType, HllPTGuard, pt_is_hll, TYPE_HLL)
 VALUE_GUARD(PrimitiveType, BinaryPTGuard, pt_is_binary, TYPE_CHAR, TYPE_VARCHAR)
 
 VALUE_GUARD(PrimitiveType, DatePTGuard, pt_is_date, TYPE_DATE)
 VALUE_GUARD(PrimitiveType, DateTimePTGuard, pt_is_datetime, TYPE_DATETIME)
 VALUE_GUARD(PrimitiveType, TimePTGuard, pt_is_time, TYPE_TIME)
+VALUE_GUARD(PrimitiveType, DecimalV1PTGuard, pt_is_decimalv1, TYPE_DECIMAL)
```

Comment:
Needn't support decimal v1. We will remove decimal v1 related codes soon

Reply:
ok